autoinstrumentation/nodejs/src/autoinstrumentation.ts

```diff
@@ -1,12 +1,34 @@
 import { getNodeAutoInstrumentations } from '@opentelemetry/auto-instrumentations-node';
 import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-grpc';
+import { alibabaCloudEcsDetector } from '@opentelemetry/resource-detector-alibaba-cloud';
+import { awsEc2Detector, awsEksDetector } from '@opentelemetry/resource-detector-aws';
+import { containerDetector } from '@opentelemetry/resource-detector-container';
+import { gcpDetector } from '@opentelemetry/resource-detector-gcp';
+import { envDetector, hostDetector, osDetector, processDetector } from '@opentelemetry/resources';
 
 import { NodeSDK } from '@opentelemetry/sdk-node';
 
 const sdk = new NodeSDK({
     autoDetectResources: true,
     instrumentations: [getNodeAutoInstrumentations()],
     traceExporter: new OTLPTraceExporter(),
+    resourceDetectors:
+        [
+            // Standard resource detectors.
+            containerDetector,
```

Comment:
Do you have a span example e.g. from Jaeger that shows which resource attributes are collected? 

Note that the operator as well injects the env var OTEL_RESOURCE_ATTRIBUTES with k8s attributes (container name, pod etc.)

Reply:
> Does this work all the time or only in Sidecar mode (as the README.md suggests)?

The OTEL_RESOURCE_ATTRIBUTES is injected into all application containers that use the instrumentation CR.

The collector CR injects the attributes as well when deployed in the sidecar mode

Reply:
> > Does this work all the time or only in Sidecar mode (as the README.md suggests)?
> 
> The OTEL_RESOURCE_ATTRIBUTES is injected into all application containers that use the instrumentation CR.
> 
> The collector CR injects the attributes as well when deployed in the sidecar mode

That's interesting, is there a full list of attributes that are injected (somewhere in the source code I assume?)